### PR TITLE
Fixed #35084 -- Recommended 'django_' prefix for reusable app modules.

### PR DESCRIPTION
**Pull request by adamchainz**
ticket-35084

### CODE REVIEW

File: docs/intro/reusable-apps.txt [hunk 1]
```diff
@@ -76,7 +76,7 @@ After the previous tutorials, our project should look like this:
             static/
                 polls/
                     images/
-                        background.gif
+                        background.png
```
Comment (adamchainz) [inline docs/intro/reusable-apps.txt:79]: I missed this in 76fda7729e4cdfec715cd92b2c80d851797b05f7

File: docs/intro/reusable-apps.txt [hunk 2]
```diff
@@ -61,9 +61,9 @@ After the previous tutorials, our project should look like this:
         manage.py
         mysite/
             __init__.py
+            asgi.py
             settings.py
             urls.py
-            asgi.py
             wsgi.py
```
Comment (adamchainz) [inline docs/intro/reusable-apps.txt:67]: Added out of order in 3930ec1bf275d17f1c36cd38b38114177b1d1565

Comment (felixxm) [reply docs/intro/reusable-apps.txt:67]: Is the order really important here? I wanted to put it next to the `wsgi.py`.

Comment (adamchainz) [reply docs/intro/reusable-apps.txt:67]: All file listings in the different tutorial steps and elsewhere in the docs are alphabetical. I see no reason to special-case `asgi.py`. I will extract these changes into a separate PR.

Comment (adamchainz) [reply docs/intro/reusable-apps.txt:67]: See #17680.

File: docs/intro/reusable-apps.txt [hunk 3]
```diff
@@ -299,14 +314,25 @@ working. We'll now fix this by installing our new ``django-polls`` package.
 
       python -m pip install --user django-polls/dist/django-polls-0.1.tar.gz
 
-#. With luck, your Django project should now work correctly again. Run the
-   server again to confirm this.
+#. Update ``mysite/settings.py`` to point to the new module name:
```
Comment (pauloxnet) [inline docs/intro/reusable-apps.txt:317]:
```suggestion
#. Update ``mysite/settings.py`` to point to the new module name::
```
I think we are using `::` for `python` code (default).

File: docs/intro/reusable-apps.txt [hunk 4]
```diff
@@ -299,14 +314,25 @@ working. We'll now fix this by installing our new ``django-polls`` package.
...
 
-#. To uninstall the package, use pip:
+   .. code-block:: python
```
Comment (pauloxnet) [inline docs/intro/reusable-apps.txt:319]:
```suggestion
```

File: docs/intro/reusable-apps.txt [hunk 5]
```diff
@@ -299,14 +314,25 @@ working. We'll now fix this by installing our new ``django-polls`` package.
...
 
-   .. code-block:: shell
+        INSTALLED_APPS = [
+            "django_polls.apps.PollsConfig",
+            ...,
+        ]
+
+#. Update ``mysite/urls.py`` to point to the new module name:
+
+   .. code-block:: python
```
Comment (pauloxnet) [inline docs/intro/reusable-apps.txt:328]:
```suggestion
#. Update ``mysite/urls.py`` to point to the new module name::
```

File: docs/intro/reusable-apps.txt [hunk 6]
```diff
@@ -121,36 +121,52 @@ Python *packaging* refers to preparing your app in a specific format that can
...
 
-#. Move the ``polls`` directory into the ``django-polls`` directory.
+#. Move the ``polls`` directory into ``django-polls`` directory, and rename it
+   to ``django_polls``.
+
+#. Edit ``django_polls/apps.py`` so that :attr:`~.AppConfig.name` refers to the
+   new module name and add :attr:`~.AppConfig.label` to give a short name for
+   the app:
+
+   .. code-block:: python
```
Comment (pauloxnet) [inline docs/intro/reusable-apps.txt:149]:
```suggestion
   the app::
```
See below.

**Comment by felixxm** [comment]
@adamchainz Thanks :+1:

File: docs/intro/reusable-apps.txt (hunk 6 above)
Comment (pauloxnet) [reply docs/intro/reusable-apps.txt:149]: @felixxm you accepted the other suggestions, but not this one. Can I ask the motivation?

Comment (felixxm) [reply docs/intro/reusable-apps.txt:149]: because I added a `caption`.

Comment (pauloxnet) [reply docs/intro/reusable-apps.txt:149]: Now it's clear. Thanks.

**Pull request merged**